Muehle_GUI.ipynb

```diff
@@ -393,17 +393,27 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "def update_status_widgets():\n",
+    "def update_status_widgets(message):\n",
     "    turn_label.value='Spieler '+ str(game.current_player)+ ' ist dran.'\n",
-    "    change_status_label('Spieler in Phase '+ str(get_player_phase(game.state, game.current_player))+'.')\n",
+    "    if(get_player_phase(game.state, game.current_player) == 1):\n",
+    "        change_status_label('Bitte einen Stein auf dem Spielfeld platzieren.')\n",
+    "    if(get_player_phase(game.state, game.current_player) == 2):\n",
+    "        change_status_label('Bitte einen Stein zum Ziehn auswählen.')\n",
```

Comment:
```suggestion
    "        change_status_label('Bitte einen Stein zum Ziehen auswählen.')\n",
```

Reply:
da muss ich die passende Stelle in make move finden